### DISCUSSION TITLE
Cannot change my sponsorship tier

### DISCUSSION THREAD

**Discussion by stardust-xs** [community/community#64518] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've recently sponsored an account that I follow. I can't seem to change my sponsorship tier. Any ideas why?

**Comment by xames3** [comment]
This could probably because the sponsor has set some pre-defined tiers for his/her account/organization and when you enroll into the program you cannot change that. You need to choose the tier that works for you and subscribe to it, either monthly or one-time.

**Reply by stardust-xs** [reply]
Sure. I got it. Thanks!

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
